R/as_curve_params.R

```diff
@@ -31,9 +45,8 @@ as_curve_params <- function(data, antigen_isos = NULL) {
     cli::cli_abort(
       class = "not curve_params",
       message = c(
-        "Can't convert {.envvar data} to curve_params.",
-        "i" = "The column(s):{.strong {.var {missing_cols}}} are missing.",
-        "x" = "You have supplied {.cls {class(data)}}."
+        "Can't convert {.arg data} to {.cls curve_params}.",
+        "x" = "The column{?s}: {.strong {.var {missing_cols}}} are missing."
```

Comment:
see https://cli.r-lib.org/reference/pluralization.html#pluralization-markup

Reply:
This is well-noted.